staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -18,6 +18,11 @@ package v1beta1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+// TODO: This does not generate to protobuf because of the interface{} - fix protobuf
+// generation to support a meta type that can accept any valid JSON.
+
+// +protobuf=false
```

Comment:
One way would be by having @nikhita's `JSON` in the external types and `interface{}` in the internal. Then we can use normal proto tags on `JSON` and custom JSON marshal/unmarshal methods. Alternatively custom marshal/unmarshal for protobuf if that exists.